paymentapi/src/main/java/fi/hel/verkkokauppa/payment/paytrail/context/PaytrailPaymentContext.java

```diff
@@ -6,11 +6,18 @@
 public class PaytrailPaymentContext {
 
     private String namespace;
+    private String internalMerchantId;
```

Comment:
merchantId  -> internalMerchantId. This is needed to get merchant specific configurations for normal merchant flow.

Reply:
Add this as comment to code 👍